src/cython/vapoursynth.pyx

```diff
@@ -1865,7 +1865,10 @@ cdef class Function(object):
 cdef Function createFunction(str name, str signature, Plugin plugin, const VSAPI *funcs):
     dct = {'__signature__': None}
     if typing is not None:
-        dct['__signature__'] = construct_signature(signature, injected=plugin.injected_arg)
+        try:
+            dct['__signature__'] = construct_signature(signature, injected=plugin.injected_arg)
+        except ValueError as e:
+            print("Constructing signature for {}.{}() failed: {}".format(plugin.namespace, name, str(e)))
```

Comment:
Use `warnings` instead of `print`. https://docs.python.org/3/library/warnings.html
Warnings can be explicitly captured. Print-calls can't.